sdk/keyvault/azure-security-keyvault-certificates/README.md

```diff
@@ -14,7 +14,7 @@ Maven dependency for the Azure Key Vault Certificate client library. Add it to y
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-certificates</artifactId>
-    <version>4.2.0-beta.4</version>
+    <version>4.2.0-beta.5</version>
```

Comment:
Is this version released?

Reply:
Please refer to [this comment](https://github.com/Azure/azure-sdk-for-java/pull/20521#discussion_r610940121).